src/ts/client/grapesjs/keymaps.ts

```diff
@@ -0,0 +1,111 @@
+import {Editor, PluginOptions} from 'grapesjs'
+import {cmdPublish} from './PublicationUi'
+import {cmdOpenFonts} from '@silexlabs/grapesjs-fonts'
+import {cmdToggleBlocks, cmdToggleLayers, cmdToggleNotifications, cmdToggleSymbols} from './index'
+import {cmdTogglePages} from './page-panel'
+import {cmdOpenSettings} from './settings'
+import {isTextOrInputField, selectBody} from '../utils'
+import {PublishableEditor} from './PublicationManager'
+
+export const cmdSelectBody = 'select-body'
+export let prefixKey = 'Shift'
+export const defaultKms = {
+  kmOpenSettings: 'Alt+S',
+  kmOpenPublish: 'Alt+P',
+  kmOpenFonts: 'Alt+F',
+  kmPreviewMode: 'tab',
+  kmLayers: prefixKey + '+L',
+  kmBlocks: prefixKey + '+A',
+  kmNotifications: prefixKey + '+N',
+  kmPages: prefixKey + '+P',
+  kmSymbols: prefixKey + '+S',
+  kmClosePanel: 'Escape'
+}
+
+// Utility functions
+
+const toggleCommand = (editor: Editor, name: string): void => {
```

Comment:
it is not very important but you probably want to use function ABCD() {} insead of const ABCD = () => {}

because arrow function makes no sense here without a context

Reply:
By the way, I see you also use arrow functions "without context" for main plugin functions (ex: project-bar.ts), do I change all of the arrow functions into "classical" functions?